docs/docs/gatsby-folder-structure.md

```diff
@@ -0,0 +1,40 @@
+## Gatsby Folder Structure:
+
+```
+/                      // Root
+|-- /.cache            // Gatsby's cache directory (created automatically)
+|-- /plugins           // (optional) To host local plugins code - link to writing plugin docs
+|-- /public            // Gatsby's output directory (created automatically)
+|-- /src               // Your project source
+    |-- /pages         // describe pages dir
+    |-- /templates     // (optional) describe templates dir
+    |-- html.js        // (optional) to customise .cache/default-html.js
```

Comment:
`customize`

Reply:
Well, actually I'd just remove the comments completely as your describing the items just right under it and hence it's a duplicate, right?